### PR TITLE
add descriptions to args in the source schema

### PR DESCRIPTION
**Pull request by bendichter**

### CODE REVIEW

File: src/neuroconv/datainterfaces/ecephys/neuroscope/neuroscopedatainterface.py [hunk 1]
```diff
@@ -101,6 +101,12 @@ class NeuroScopeRecordingInterface(BaseRecordingExtractorInterface):
     associated_suffixes = (".dat", ".xml")
     info = "Interface for converting NeuroScope recording data."
 
+    @classmethod
+    def get_source_schema(self) -> dict:
+        source_schema = super().get_metadata_schema()
```
Comment (CodyCBakerPhD) [inline src/neuroconv/datainterfaces/ecephys/neuroscope/neuroscopedatainterface.py:106]:
```suggestion
        source_schema = super().get_source_schema()
```

File: src/neuroconv/datainterfaces/ecephys/neuroscope/neuroscopedatainterface.py [hunk 2]
```diff
@@ -188,6 +194,12 @@ class NeuroScopeLFPInterface(BaseLFPExtractorInterface):
 
     ExtractorName = "NeuroScopeRecordingExtractor"
 
+    @classmethod
+    def get_source_schema(self) -> dict:
+        source_schema = super().get_metadata_schema()
```
Comment (CodyCBakerPhD) [inline src/neuroconv/datainterfaces/ecephys/neuroscope/neuroscopedatainterface.py:199]:
```suggestion
        source_schema = super().get_source_schema()
```

File: src/neuroconv/datainterfaces/ecephys/neuroscope/neuroscopedatainterface.py [hunk 3]
```diff
@@ -240,6 +252,19 @@ class NeuroScopeSortingInterface(BaseSortingExtractorInterface):
     associated_suffixes = (".res", ".clu", ".res.*", ".clu.*", ".xml")
     info = "Interface for converting NeuroScope recording data."
 
+    @classmethod
+    def get_source_schema(self) -> dict:
+        source_schema = super().get_metadata_schema()
```
Comment (CodyCBakerPhD) [inline src/neuroconv/datainterfaces/ecephys/neuroscope/neuroscopedatainterface.py:257]:
```suggestion
        source_schema = super().get_sourcec_schema()
```

File: src/neuroconv/datainterfaces/ecephys/mearec/mearecdatainterface.py [hunk 4]
```diff
@@ -16,6 +16,12 @@ class MEArecRecordingInterface(BaseRecordingExtractorInterface):
     associated_suffixes = (".h5",)
     info = "Interface for MEArec recording data."
 
+    @classmethod
+    def get_source_schema(cls) -> dict:
+        source_schema = super().get_source_schema()
+        source_schema["properties"]["folder_path"]["description"] = "Path to the MEArec .h5 file."
```
Comment (CodyCBakerPhD) [inline src/neuroconv/datainterfaces/ecephys/mearec/mearecdatainterface.py:22]:
```suggestion
        source_schema["properties"]["file_path"]["description"] = "Path to the MEArec .h5 file."
```

**Comment by CodyCBakerPhD** [comment]
Looks good - had to change a few invalid entries but all passing now

**Review by CodyCBakerPhD — APPROVED** [review]

**Comment by garrettmflynn** [comment]
Just FYI, something is breaking on the GUIDE when you try to generate all the schemas using `generateInterfaceSchema.py`

**Comment by garrettmflynn** [comment]
Disregard the last message. Looks great!

**Pull request merged**